packages/website/src/components/editor/useSandboxServices.ts

```diff
@@ -145,7 +145,10 @@ export const useSandboxServices = (
         model.dispose();
       }
     };
-  }, [props.ts, colorMode, props.jsx, onLoaded]);
+    // colorMode can't be reactive here, this is going to cause crash while switching dark/light mode
+    // updating of colorMode is handled in LoadedEditor
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.ts, props.jsx, onLoaded]);
```

Comment:
we have to make sure that this value is non reactive,

@JoshuaKGoldberg if you don't like eslint disable we can save this value to state and never mutate it

```
const [darkMode] = setState(() => colorMode)
```